src/widgets/ReportTable/ReportTable.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
```

Comment:
this is for the `renderItem` call? Theres some info here about that if you're interested https://github.com/yannickcr/eslint-plugin-react/issues/2325

Reply:
Yes, I have a problem with that indeed. If I declared them then `es-lint` says I'm not using them 😅